docker/vm_boot_images/config/tensorflow-requirements.txt

```diff
@@ -8,7 +8,6 @@ pydicom==1.2.2
 hyperopt==0.1.2
 seaborn==0.9.0
 scikit-image
-scikit-survival
```

Comment:
This one has to go, right? Was it conflicting with some GitHub action?

Reply:
Yeah wanted numpy 2.20 lol smh.